pkg/resource/deploy/source_eval.go

```diff
@@ -925,3 +939,14 @@ func (g *readResourceEvent) AdditionalSecretOutputs() []resource.PropertyKey {
 func (g *readResourceEvent) Done(result *ReadResult) {
 	g.done <- result
 }
+
+func generateTimeoutInSeconds(timeout string) float64 {
+	duration, err := time.ParseDuration(timeout)
+	if err != nil {
+		logging.V(5).Infof("ResourceMonitor.RegisterResource unable to parse customTimeout Value %s",
```

Comment:
I think we need to be surfacing this error to the user.

Reply:
done!